docs/index.rst

```diff
@@ -40,39 +42,67 @@ resources when not actively computing.
 
 .. code-block:: python
 
-   cluster.adapt()
+   cluster.adapt(minimum=1, maximum=100)
 
+.. toctree::
+   :maxdepth: 1
+   :hidden:
 
-History
--------
+   install.rst
+   examples.rst
+   history.rst
+   api.rst
 
-This package came out of the `Pangeo <https://pangeo-data.github.io/>`_
-collaboration and was copy-pasted from a live repository at
-`this commit <https://github.com/pangeo-data/pangeo/commit/28f86b9c836bd622daa14d5c9b48ab73bbed4c73>`_.
-Unfortunately, development history was not preserved.
+How this works
+--------------
 
-Original developers include the following:
+This creates a Dask Scheduler in the Python process where the cluster object
+is instantiated:
 
--  `Jim Edwards <https://github.com/jedwards4b>`_
--  `Joe Hamman <https://github.com/jhamman>`_
--  `Matthew Rocklin <https://github.com/mrocklin>`_
+.. code-block:: python
 
-**Getting Started**
+   cluster = PBSCluster(processes=18,
+                        threads=4,
+                        memory="6GB",
+                        project='P48500028',
+                        queue='premium',
+                        resource_spec='select=1:ncpus=36:mem=109G',
+                        walltime='02:00:00')  # <-- scheduler started here
 
-* :doc:`install`
-* :doc:`examples`
+When you ask for more workers, such as with the ``scale`` command
 
-.. toctree::
-   :maxdepth: 1
-   :hidden:
-   :caption: Getting Started
+.. code-block:: python
 
-   install.rst
-   examples.rst
+   cluster.scale(10)
 
-.. toctree::
-   :maxdepth: 2
-   :hidden:
-   :caption: Contents:
+The cluster generates a traditional job script and submits that an appropriate
+number of times to the job queue.  You can see the job script that it will
+generate as follows:
 
-   api.rst
+.. code-block:: python
+
+   >>> print(cluster.job_script())
+
+.. code-block:: bash
+
+   #!/bin/bash
+
+   #PBS -N dask-worker
+   #PBS -q premium
+   #PBS -A P48500028
+   #PBS -l select=1:ncpus=36:mem=109G
+   #PBS -l walltime=02:00:00
+
+   /home/mrocklin/Software/anaconda/bin/dask-worker tcp://127.0.1.1:43745
+   --nthreads 4 --nprocs 18 --memory-limit 6GB --name dask-worker-3
+   --death-timeout 60
+
+Each of these jobs are sent to the job queue independently and, once that job
+starts, a dask-worker process will start up and connect back to the scheduler
+running within this process.
+
+If the job queue is busy then it's possible that the workers will take a while
+to get through or that not all of them arrive.  Operating with fewer workers
+than expected is OK, Dask can elastically scale.  We recommend submitting small
```

Comment:
I don't know if I would _recommend_ small jobs as it can also create gaps in the job queue cluster use :) I would rather say "A solution for interactive work is" or something like that... Short wall time is perfectly OK though.

Reply:
OK, I've changed the wording to be more of an observation.  I'm also happy to remove it entirely.

Reply:
Thanks for the modification, I am happy with that. You've altered your message though, but if it's OK for you, lets stick with this.

Reply:
> You've altered your message though, but if it's OK for you, lets stick with this.

￼I have.  Now it's our message.  I like it better this way :)